cmd/admin-handlers-config-kv.go

```diff
@@ -201,6 +208,19 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
+	// Check if callhome getting enabled, if so set the subnet proxy value to logger_webhook's subnet proxy
+	// Do this only if logger_wenhook's subnet proxy is not yet set
+	callhomeCfg := result.Cfg[config.CallhomeSubSys]["_"]
+	if result.SubSys == config.CallhomeSubSys && callhomeCfg.Get(callhome.Enable) == "on" {
+		loggerWebhookSubnetProxy := result.Cfg[config.LoggerWebhookSubSys][config.SubnetSubSys].Get(logger.Proxy)
```

Comment:
Even though the value of `config.SubnetSubSys` is `subnet` which needs to be used here, it is not the "subsystem", but the name of the webhook target whose proxy config we're trying to get. Maybe you should introduce a new constant and use it here, say `subnet.LoggerWebhookName`

Reply:
Ack